scripts/memory/base.py

```diff
@@ -6,8 +6,10 @@
 
 def get_ada_embedding(text):
     text = text.replace("\n", " ")
-    return openai.Embedding.create(input=[text], model="text-embedding-ada-002")["data"][0]["embedding"]
-
+    try:
+        return openai.Embedding.create(input=[text], model="text-embedding-ada-002")["data"][0]["embedding"]
+    except Exception as e:
+        print("Error getting embedding: ", e)
```

Comment:
We could do with returning something to the AI here to let them know their command didn't work and that they should try again.